base/src/constants.rs

```diff
@@ -16,3 +16,12 @@ pub(crate) const LAST_ROW: i32 = 1_048_576;
 // NaiveDate::from_ymd(1900, 1, 1).num_days_from_ce() - 2
 // The 2 days offset is because of Excel 1900 bug
 pub(crate) const EXCEL_DATE_BASE: i32 = 693_594;
+
+// Excel can handle dates until the year 0000-01-01
+// However, it uses a different numbering scheme for dates
+// that are before 1900-01-01.
+// So for now we will simply not support dates before 1900-01-01.
+pub(crate) const EXCEL_DATE_MIN: i32 = 2;
```

Comment:
Could you rename it to `DATE_SERIAL_NUMBER_MINIMUM`? and also the other one.
Note that they have different units from the (also a bit ill-named) `EXCEL_DATE_BASE`